mediaflow_proxy/extractors/dlhd.py

```
@@ -1,588 +1,323 @@
+            # Determina endpoint in base al dominio dello stream
+            endpoint = "hls_key_proxy"
+
+            logger.info(f"Using lovecdn.ru stream with endpoint: {endpoint}")
+            
+            return {
+                "destination_url": stream_url,
+                "request_headers": stream_headers,
+                "mediaflow_endpoint": endpoint,
+            }
+            
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)